production-rules/DST_STREAMING_EVENT_run2pp_ana435_2024p007.yaml

```diff
@@ -0,0 +1,202 @@
+PHYS_DST_SINGLE_STREAMING_EVENT_run2pp:
```

Comment:
This is just a test file... and an early one at that.  We should probably just remove it.